gulpfile.js

```diff
@@ -174,7 +196,7 @@ gulp.task('demo-push', function() { return gulp.src(PATHS.demoDist).pipe(ghPages
 gulp.task('clean', ['clean:build', 'clean:tests', 'clean:demo', 'clean:demo-cache']);
 
 gulp.task('build', function(done) {
-  runSequence('lint', 'enforce-format', 'ddescribe-iit', 'test', 'clean:build', 'cjs', 'esm', 'umd', done);
+  runSequence('lint', 'enforce-format', 'ddescribe-iit', 'test', 'clean:build', 'cjs', 'esm', 'umd', 'npm', done);
```

Comment:
Let's create a release task please, and if it can double check that you want to publish, better. This task is used as a command to see that everything is working and there is no error.